Enhancement/10248 bulk edit setting

ankitrox opened the pull request:
## Summary

<!-- Please reference the issue this PR addresses in the following list. -->
Addresses issue:

- #10248 

## Relevant technical choices

<!-- Please describe your changes. -->

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 5.2 and PHP 7.4.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.
- [ ] Ensure there are no unexpected significant changes to file sizes.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

nfmohit commented on includes/Modules/Reader_Revenue_Manager/Admin_Post_List.php:
```diff
@@ -0,0 +1,241 @@
+				$product_id      = ( false !== $separator_index ) ? substr( $post_product_id, $separator_index + 1 ) : $post_product_id;
+
+				/* translators: %s: Product ID */
+				echo esc_html( sprintf( __( 'Use "%s"', 'google-site-kit' ), $product_id ) );
```
According to the ACs, the value here should solely use the product ID, without "_Use_".

nfmohit reviewed (changes requested):
Brilliant work here, thank you @ankitrox!

I've added a few comments for your consideration. Please let me know if you have any questions, thank you!

ankitrox commented on tests/phpunit/integration/Modules/Reader_Revenue_Manager/Admin_Post_ListTest.php:
```diff
@@ -0,0 +1,175 @@
+		$this->assertNotEquals( 'ABCD:basic', $this->post_product_id->get( $post_id ) );
+	}
+
+	/**
+	 * @group 10up
+	 */
```
Oops! Sorry, it was left there by mistake. Remove it now.

ankitrox commented:
Thanks for reviewing the PR @nfmohit and adding your feedback. I have addressed the feedback and pushed the changes.

Assigning it to you to review these changes.

nfmohit reviewed (approved):
LGTM, nice work, @ankitrox!
